README.md

```diff
@@ -248,16 +248,43 @@ func application(_ application: UIApplication, continue userActivity: NSUserActi
 	return [IterableAPI handleUniversalLink:userActivity.webpageURL];
 }
 ```
+
+### Deferred deep linking
+
+Deferred deep linking comes into play when a user of your app does not have your app installed in their iPhone/iPad. See definition of deferred deep linking [here](#https://en.wikipedia.org/wiki/Deferred_deep_linking). If you enable deferred deeplinking in IterableSDK, when the user clicks on a deep link in an email, the user is first sent to the App Store. After installing your app from the App Store, the user is taken to a specific location within your app when they launch it for the first time. 
+
+#### Enabling deferred deep linking
+
+Set `IterableConfig.checkForDeferredDeeplink = true` to enable deferred deep linking for IterableSDK.
 	
-#### In-app messages
+### In-app messages
+
+If you are already using in-app messages with IterableSDK, please check out the [migration section](#Migrating-in-app-messages-from-the-previous-version-of-the-SDK). 
 
-If you are already using in-app messages with IterableSDK, please check out the [migration section](#Migrating-in-app-messages-from-previous-version-of-SDK).
+In-app messages are handled via silent push messages from the server. When your application receives a silent push, you need to call IterableSDK in your AppDelegate as in the following code:
```

Comment:
- You don't _need_ to do this, right? If you don't, the SDK will default to syncing the queue and showing the newest message? Just want to be sure we don't imply that users have to do something they don't.

Reply:
Oh, I thought this all happened by default, but was customizable as necessary.

Reply:
Yes we need to hook up silent push for ddl to work. I may have overlooked that earlier and caused some confusion.